console-subscriber/src/aggregator.rs

```diff
@@ -427,6 +539,10 @@ impl<T> TaskData<T> {
     fn all(&self) -> impl Iterator<Item = (&span::Id, &T)> {
         self.data.iter().map(|(id, (data, _))| (id, data))
     }
+
+    fn find(&self, id: &span::Id) -> Option<&T> {
```

Comment:
nit, take it or leave it: might have called this `get` to keep it consistent with the `HashMap` method? not a blocker